tests/pytorch_tests/model_tests/feature_models/mixed_precision_activation_test.py

```diff
@@ -112,7 +112,7 @@ def compare(self, quantized_models, float_model, input_x=None, quantization_info
 class MixedPrecisionActivationSearch4BitFunctional(MixedPrecisionActivationBaseTest):
     def __init__(self, unit_test):
         super().__init__(unit_test)
-        self.expected_config = [1, 4, 4, 1]
+        self.expected_config = [2, 5, 5, 1]
```

Comment:
did you verify that this result make sense? (as far as possible, I mean by looking at the max cut result and observing the network structure)?